epitome/test/models_test.py

```diff
@@ -116,7 +116,7 @@ def test_model_two_similarity_assays(self):
 		# should train a model without using DNAse
 		eligible_targets = ['CTCF', 'RAD21', 'CEBPB']
 
-		dataset = EpitomeDataset(targets = eligible_targets, similarity_targets = ['DNase', 'H3K27ac'])
+		dataset = EpitomeDataset(targets = eligible_targets, similarity_targets = ['DNase', 'H3K27ac'], data_dir=self.epitome_test_path)
```

Comment:
also set `assembly = "test"` for all of these cases

Reply:
Done